llama_toolchain/cli/download.py

```diff
@@ -178,6 +178,9 @@ class ModelEntry(BaseModel):
     model_id: str
     files: Dict[str, str]
 
+    class Config:
```

Comment:
why is this needed in this PR ?

Reply:
it is technically not needed. it's an independent warning-fix. I can separately commit it